utils/rpms/daos.spec

```diff
@@ -128,7 +129,8 @@ PREFIX="%{?_prefix}"
 sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/TESTING/.build_vars.*
 mv %{?buildroot}%{_prefix}/lib{,64}
 #mv %{?buildroot}/{usr/,}etc
-mkdir -p %{?buildroot}/%{daoshome}
+mkdir -p %{?buildroot}/%{_exec_prefix}/lib/%{name}
+mv %{?buildroot}%{_prefix}/lib64/daos %{?buildroot}/%{_exec_prefix}/lib/
```

Comment:
Hrm.  I'm surprised this works.  On line 132 you create `/usr/lib/daos/` and then on line 133 you move `/usr/lib64/daos` to `/usr/lib/`.  I'd expect you to get an error from the `mv`.  But it doesn't seem to.  It seems to happly replace an empty directory.  Learned something new.